blt/middleware/ip_restrict.py

```
@@ -1,5 +1,7 @@
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)